[2020-04-22T13:40:55.239650+02:00] console.ERROR: Error thrown while running command "'server:run'". Message: "Command "server:run" is not defined.  Did you mean one of these?     server:dump     server:log" {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    server:dump\n    server:log at /home/maxime/Documents/Crowding/mon-super-projet/vendor/symfony/console/Application.php:644)","command":"'server:run'","message":"Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    server:dump\n    server:log"} []
[2020-04-22T13:42:09.259794+02:00] console.ERROR: Error thrown while running command "'server:run'". Message: "Command "server:run" is not defined.  Did you mean one of these?     server:dump     server:log" {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    server:dump\n    server:log at /home/maxime/Documents/Crowding/mon-super-projet/vendor/symfony/console/Application.php:644)","command":"'server:run'","message":"Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    server:dump\n    server:log"} []
[2020-04-22T13:42:19.681421+02:00] console.ERROR: Error thrown while running command "'server:run'". Message: "Command "server:run" is not defined.  Did you mean one of these?     server:dump     server:log" {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    server:dump\n    server:log at /home/maxime/Documents/Crowding/mon-super-projet/vendor/symfony/console/Application.php:644)","command":"'server:run'","message":"Command \"server:run\" is not defined.\n\nDid you mean one of these?\n    server:dump\n    server:log"} []
[2020-04-22T13:42:56.186728+02:00] php.DEBUG: Warning: filemtime(): stat failed for /home/maxime/Documents/mon-super-projet/config/routes/dev/framework.yaml {"exception":{"Symfony\\Component\\ErrorHandler\\Exception\\SilencedErrorContext":{"severity":2,"file":"/home/maxime/Documents/Crowding/mon-super-projet/vendor/symfony/config/Resource/FileResource.php","line":65,"trace":[{"file":"/home/maxime/Documents/Crowding/mon-super-projet/vendor/symfony/config/Resource/SelfCheckingResourceChecker.php","line":34,"function":"isFresh","class":"Symfony\\Component\\Config\\Resource\\FileResource","type":"->"}],"count":1}}} []
[2020-04-22T13:42:56.300486+02:00] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /"" at /home/maxime/Documents/Crowding/mon-super-projet/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /home/maxime/Documents/Crowding/mon-super-projet/vendor/symfony/http-kernel/EventListener/RouterListener.php:136)\n[previous exception] [object] (Symfony\\Component\\Routing\\Exception\\NoConfigurationException(code: 0):  at /home/maxime/Documents/Crowding/mon-super-projet/vendor/symfony/routing/Matcher/Dumper/CompiledUrlMatcherTrait.php:182)"} []
[2020-04-22T13:42:56.364183+02:00] request.INFO: Matched route "_wdt". {"route":"_wdt","route_parameters":{"_route":"_wdt","_controller":"web_profiler.controller.profiler::toolbarAction","token":"a14dd6"},"request_uri":"http://localhost:8000/_wdt/a14dd6","method":"GET"} []
